CHANGELOG.md

```diff
@@ -25,4 +27,4 @@ Change log
 # v2.0.0-beta.8 - 2017-01-27
 
 * Update meta-resin to v2.0-beta.8 [Andrei]
-* Initial commit on Yocto 2.1 [Florin]
\ No newline at end of file
+* Initial commit on Yocto 2.1 [Florin]
```

Comment:
I think emacs added a newline at the end of the document as it assumes any document called "changelog" is a GNU changelog! I should really turn that off...